frontend/device/pocketbook/device.lua

```diff
@@ -103,6 +109,15 @@ function PocketBook:initNetworkManager(NetworkMgr)
     end
 end
 
+function PocketBook:getSoftwareVersion()
+    return ffi.string(inkview.GetSoftwareVersion())
```

Comment:
@poire-z Is that a good idea wrt ffi?

Reply:
Seems like the way to do it, lots of them in ffi/util.lua when getting char* from C calls.

Reply:
It shouldn't matter either way. Using a named local for clarity should still get cleaned up. Alright, I'll merge this then.